channeldb/error.go

```diff
@@ -128,6 +128,10 @@ var (
 	// channel with a channel point that is already present in the
 	// database.
 	ErrChanAlreadyExists = fmt.Errorf("channel already exists")
+
+	// ErrNoOutpointBucket is returned when a outpoint bucket does not
+	// exist.
+	ErrNoOutpointBucket = fmt.Errorf("outpoint bucket has not been created")
```

Comment:
this error should be possible due to the TLB migration. doesn't look like it's used anywhere?

Reply:
It won't be used if we use `CreateTopLevelBucket`. See comment here: https://github.com/lightningnetwork/lnd/pull/4642#issuecomment-707180314

Reply:
It's used now

Reply:
looks like the only places it's used are impossible to reach?

Reply:
My logic is that if it's possible for `ReadWriteBucket` to return nil we should handle it

Reply:
the function returns that, but it is impossible due to our other constraints. if `nil` somehow gets returned then we will panic, which is what we want because something is very wrong. this is how we've handled this case in every other TLB migration.

Reply:
done